### ISSUE TITLE
ClassNotFoundException: Attempted to load class "Utf8"

### ISSUE THREAD

**Issue by xchs** [inspiredminds/contao-sibling-navigation#1]
I get the following error message when I try to include the front end module in the article:
```
ErrorException: Uncaught Symfony\Component\Debug\Exception\ClassNotFoundException: Attempted to load class "Utf8" from namespace "InspiredMinds\ContaoSiblingNavigation\Module".
Did you forget a "use" statement for "Patchwork\Utf8"? in /vendor/inspiredminds/contao-sibling-navigation/src/Module/SiblingNavigationEvents.php:44
Stack trace:
#0 /vendor/contao/core-bundle/src/Resources/contao/elements/ContentModule.php(78): InspiredMinds\ContaoSiblingNavigation\Module\SiblingNavigationEvents->generate()
#1 /vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php(483): Contao\ContentModule->generate()
#2 /var/cache/prod/contao/dca/tl_content.php(1266): Contao\Controller::getContentElement(Object(Contao\ContentModel))
#3 /vendor/contao/core-bundle/src/Resources/contao/driv
#1 vendor/inspiredminds/contao-sibling-navigation/src/Module/SiblingNavigationEvents.php(44): handleFatalError
#0 vendor/sentry/sentry/lib/Raven/ErrorHandler.php(0): null
```

```
Error: Class 'InspiredMinds\ContaoSiblingNavigation\Module\Utf8' not found
#18 vendor/inspiredminds/contao-sibling-navigation/src/Module/SiblingNavigationEvents.php(44): generate
#17 vendor/contao/core-bundle/src/Resources/contao/elements/ContentModule.php(78): generate
#16 vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php(483): getContentElement
#15 var/cache/prod/contao/dca/tl_content.php(1266): addCteType
#14 vendor/contao/core-bundle/src/Resources/contao/drivers/DC_Table.php(4461): parentView
#13 vendor/contao/core-bundle/src/Resources/contao/drivers/DC_Table.php(351): showAll
#12 vendor/contao/core-bundle/src/Resources/contao/classes/Backend.php(612): getBackendModule
#11 vendor/contao/core-bundle/src/Resources/contao/controllers/BackendMain.php(160): run
#10 vendor/contao/core-bundle/src/Controller/BackendController.php(48): mainAction
#9 vendor/symfony/http-kernel/HttpKernel.php(149): handleRaw
#8 vendor/symfony/http-kernel/HttpKernel.php(66): handle
#7 vendor/symfony/http-kernel/Kernel.php(188): handle
#6 vendor/symfony/http-kernel/HttpCache/SubRequestHandler.php(85): handle
#5 vendor/symfony/http-kernel/HttpCache/HttpCache.php(448): forward
#4 vendor/symfony/framework-bundle/HttpCache/HttpCache.php(57): forward
#3 vendor/symfony/http-kernel/HttpCache/HttpCache.php(238): pass
#2 vendor/symfony/http-kernel/HttpCache/HttpCache.php(183): handle
#1 vendor/friendsofsymfony/http-cache/src/SymfonyCache/EventDispatchingHttpCache.php(98): handle
#0 web/app.php(54): null
```

To reproduce:
* Create e.g. a new _Event sibling navigation_ front end module
* Insert the module in the article

[Contao 4.6.6]

**Comment by fritzmg** [comment]
Thx :) released in `1.1.4`

**Comment by xchs** [comment]
Thanks a lot.
